src/test/java/kitchenpos/application/fixture/MenuFixture.java

```diff
@@ -0,0 +1,51 @@
+package kitchenpos.application.fixture;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Objects;
+import java.util.UUID;
+import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
+import kitchenpos.domain.MenuProduct;
+
+public record MenuFixture(UUID id, String 메뉴명, BigDecimal 메뉴가격,
+                          MenuGroup 메뉴그룹, boolean 노출여부, List<MenuProduct> 메뉴구성품) {
+
+    private static final String DEFAULT_MENU_NAME = "황금 올리브 치킨";
+    private static final String DEFAULT_MENU_PRICE = "20000";
```

Comment:
문자열 상수를 파라미터로 전달할때마다 매번 BigDecimal을 생성하기보다 
미리 BigDecimal 상수로 선언해서 직접 넘기는게 좀 더 효율적이지 않을까요? 🤔 
```suggestion
    private static final BigDecimal DEFAULT_MENU_PRICE = BigDecimal.valueOf(20_000);
```

Reply:
불필요하게 비용이 발생할 수 있는 코드네요 ㅎㅎ피드백 감사합니다